### ISSUE TITLE
Bower npm dependencies

### ISSUE THREAD

**Issue by spaesani** [bower/bower#2333]
Looking at the Bower readme install there is mention of "Bower depends on .. and npm"

How extensive are these dependencies?

I am dropping Bower into  node/deps and removing node/deps/npm for a build to default bower as the node pm.

**Comment by sheerun** [comment]
Currently it's preferred distribution method. None otherwise

**Comment by spaesani** [comment]
Excuse me?  

--
spaesani@mail.com Thursday, 21 July 2016, 10:20AM -04:00 from Adam Stankiewicz  notifications@github.com :

>Currently it's preferred distribution method. None otherwise
>

**Comment by spaesani** [comment]
The question pertains to the extent as in amount. Not preferences.
--
spaesani@mail.com Thursday, 21 July 2016, 10:20AM -04:00 from Adam Stankiewicz  notifications@github.com :

>Currently it's preferred distribution method. None otherwise
>

**Comment by spaesani** [comment]
I just had,a scons look at the license.

Doesn't day "None otherwise" anywhere. In fact it says quite the opposite. 

Does Adam Stankiewicz 's response here speak for,all the contributors past and present or do I only need to pull his "contributions" out of the clone?

"None otherwise" ?? Not on.my machine.

**Comment by spaesani** [comment]
I'll clone back 6 montjs or so prior to Adam's commits...

**Comment by spaesani** [comment]
I just had,a scons look at the license.

Doesn't day "None otherwise" anywhere. In fact it says quite the opposite. 

Does Adam Stankiewicz 's response here speak for,all the contributors past and present or do I only need to pull his "contributions" out of the clone?

"None otherwise" ?? Not on.my machine.

**Comment by spaesani** [comment]
I just had,a scons look at the license.

Doesn't day "None otherwise" anywhere. In fact it says quite the opposite. 

Does Adam Stankiewicz 's response here speak for,all the contributors past and present or do I only need to pull his "contributions" out of the clone?

"None otherwise" ?? Not on.my machine.

**Comment by spaesani** [comment]
.. checkout @{ ?? jan 1 2016 
should be clean..

**Comment by BenMann** [comment]
@spaesani All @sheerun said is that Bower relies on npm solely for distribution (read: Bower is installed via npm). Other than that it doesn't depend on npm. Furthermore I cannot make sense of anything that follows (License? Pull contributions out of clone..?!)

However I consider the original question sufficiently answered and I'm going to close this issue.. If you feel otherwise please let us know and be more specific about the actual question. 🍰

**Comment by spaesani** [comment]
I got.it....
"None otherwise " pertains to the dependencies
as in no other dependencies asides from a distribution method.
I had read that as meaning no other distribution method is allowed. Thus my rant about the license etc..
:)
In any case I built node without the npm dependency and left npm as an optional module.
sorry bout the misintepretation
sp

--
spaesani@mail.com Thursday, 21 July 2016, 00:25PM -04:00 from  spaesani@mail.com :

>The question pertains to the extent as in amount. Not preferences.
>--
>spaesani@mail.com Thursday, 21 July 2016, 10:20AM -04:00 from Adam Stankiewicz  notifications@github.com :
>
>>Currently it's preferred distribution method. None otherwise
>>
